scraper/src/github.py

```diff
@@ -116,6 +124,8 @@ def parse_event(self, event, event_time):
                 )
 
         elif event["type"] == "PullRequestEvent":
+            pr_body = event["payload"]["pull_request"]["body"]
+            no_of_linked_issues = self.parse_linked_issues(pr_body)
             if event["payload"]["action"] == "opened":
```

Comment:
Problem with this approach is that, we parse the number of linked issues for PR open events.

But during the lifecycle of a PR, we may add/remove the issues linked with the PR before it gets merged. Those wouldn't be reflected here...


We can scale points for merged pr's alone, not for pr's opened.